autoload/go/list.vim

```diff
@@ -13,10 +15,13 @@ function! go#list#Window(listtype, ...) abort
   " location list increases/decreases, cwindow will not resize when a new
   " updated height is passed. lopen in the other hand resizes the screen.
   if !a:0 || a:1 == 0
-    if l:listtype == "locationlist"
-      lclose
-    else
-      cclose
+    let autoclose_window = get(g:, 'go_list_autoclose', 1)
+    if autoclose_window
```

Comment:
I think this should work as well:

```viml
if get(g:, 'go_list_autoclose', 1)
 ...
endif
```

 no need to create the intermediate `autoclose_window` variable :)